src/diffusers/pipeline_utils.py

```diff
@@ -463,6 +463,28 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 user_agent["custom_pipeline"] = custom_pipeline
             user_agent = http_user_agent(user_agent)
 
+            info = model_info(
```

Comment:
can we wrap all of this into a function, called `is_compatible_with_safetensors()`? 

And then just do something like:

```python
if is_safetensors_available() and is_compatible_with_safetensors():
    ignore_patterns.append("*.bin")
```

Reply:
We can.

I'd rather leave `model_info` on its own though.

It's a network call, and function names like `is_compatible` seem innocuous and fast (which doing network operations is not :))